frontend/www/js/omegaup/problem/new.ts

```diff
@@ -0,0 +1,44 @@
+import { OmegaUp } from '../omegaup';
+import { types } from '../api_types';
+import T from '../lang';
+import Vue from 'vue';
+import problem_New from '../components/problem/Form.vue';
+import * as ui from '../ui';
+import * as api from '../api_transitional';
+
+OmegaUp.on('ready', () => {
+  const payload = types.payloadParsers.ProblemFormPayload('payload');
+  const problemNew = new Vue({
+    el: '#problem-new',
+    render: function(createElement) {
+      return createElement('omegaup-problem-new', {
+        props: {
+          data: this.data,
```

Comment:
```suggestion
          data: payload,
```

como no se está modificando el payload una vez que se hace render, no es necesario L37-L39 ni esto.

Reply:
Listo